### PR TITLE
Adding transaction lines

### PR DESCRIPTION
**Pull request by YoniGabe**
Adding transaction lines, needed for objects tests

### CODE REVIEW

File: entities/transaction_lines.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { NestedObject } from './base';
+
+export interface TransactionLines {
+    InternalID?: number;
```
Comment (Chasky-H) [inline entities/transaction_lines.ts:4]: You are missing a bunch of fields.
Also you need to add:
``` typescript
[key: string]: any
```
for TSA's

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pepperi-addons/papi-sdk",
-  "version": "1.18.0",
+  "version": "1.18.1",
```
Comment (Chasky-H) [inline package-lock.json:3]: Should be a minor bump, not a patch. See Readme.

File: entities/transaction_lines.ts (hunk 1 above)
Comment (Chasky-H) [reply entities/transaction_lines.ts:4]: See `activity.ts`

**Review by Chasky-H — CHANGES_REQUESTED** [review]

**Comment by OrenVilderman** [comment]
Hi, Please remove this, its no longer needed, I added it today, and tomorrow i hope we can marge

**Pull request closed without merging**